client/src/components/graph/graph.tsx

```diff
@@ -930,6 +984,22 @@ class Graph extends React.Component<GraphProps, GraphState> {
           onDoubleClick={this.handleCanvasEvent}
           onWheel={this.handleCanvasEvent}
         />
+        {testImageSrc && (
```

Comment:
Checks if the imageSrc is not null and then populates an `<img/>` with the string